doc/index.rst

```diff
@@ -69,3 +67,5 @@ c-lightning Documentation
    lightning-waitinvoice <lightning-waitinvoice.7.md>
    lightning-waitsendpay <lightning-waitsendpay.7.md>
    lightning-withdraw <lightning-withdraw.7.md>
+   lightningd <lightningd.8.md>
```

Comment:
But `lightningd` is before `lightningd-*` in alphabetic order ?

Reply:
Yes, this is why `lightningd` appears before `lightningd-config`. Notice that the earlier entries are for `lightning-...` (no trailing `d`).

Reply:
Oops, tricked by the trailing `d`.